plugins/00_dokku-standard/commands

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
-source "$PLUGIN_PATH/common/functions"
-source "$PLUGIN_PATH/config/functions"
+source "$PLUGIN_CORE_AVAILABLE_PATH/common/functions"
+source "$PLUGIN_CORE_AVAILABLE_PATH/config/functions"
```

Comment:
I just want to use a custom plugin in some cases but fall back to core if they dont exist. I'll update this pr to ensure if it's not common, then we use `$PLUGIN_AVAILABLE_PATH`.